portfolio/src/main/webapp/index.html

```diff
@@ -111,25 +104,51 @@ <h6 class="card-subtitle text-muted">Technologies Used: C++, Arduino </h6>
           <div class="card mb-4 shadow-sm">
              <img class="ctr" style="text-align:center;height:225x; width:325px;" src="img/image.png"/>
             <div class="card-body">
-              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters into a single website where students can filter and sign up for events.</p>
+              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters
+                   into a single website where students can filter and sign up for events.</p>
               <h6 class="card-subtitle text-muted">Technologies Used: Django (Python), HTML/CSS, JS</h6>
               <div class="d-flex justify-content-between align-items-center">
-                <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" <button type="button" class="btn btn-sm btn-outline-secondary">Link</button></a>
+                <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" <button type="button" class="btn btn-sm btn-outline-secondary">
```

Comment:
Tags here are messed up somehow, you have `<a <button></button></a>`

Also note that there's usually no reason to put a button in an href, it's possible to style a href to act as a button (which is typically what you do when you want a button that points to a link rather than peforming some action.

Reply:
I used a button tag and onclick, I don't know if this is the method you had in mind.